tests/data/archives/~Jackson_Upton9.jsonl

```diff
@@ -1,2 +1,2 @@
-{"id":"8g8LMXYD","content":{"title":"Remov intralum trach FB","abstract":"Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Proin risus. Praesent lectus."}}
-{"id":"ssOvsrfJ","content":{"title":"Video/radio eeg monitor","abstract":"Duis bibendum. Morbi non quam nec dui luctus rutrum. Nulla tellus.\n\nIn sagittis dui vel nisl. Duis ac nibh. Fusce lacus purus, aliquet at, feugiat non, pretium quis, lectus.\n\nSuspendisse potenti. In eleifend quam a odio. In hac habitasse platea dictumst."}}
+{"id":"8g8LMXYD", "mdate": 1234567890, "content":{"title":"Remov intralum trach FB","abstract":"Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Proin risus. Praesent lectus."}}
```

Comment:
shouldn't be here `tmdate`?

Reply:
The key name is `mdate` but the value it gets is the `tmdate` from the publication.